api/src/test/java/uk/gov/dhsc/htbhf/claimant/communications/UpcomingBirthdayEmailHandlerTest.java

```diff
@@ -143,99 +153,76 @@ void shouldSendEmailForChildTurnsOneInFirstWeekOfNextCycle() {
+                payload.getEmailPersonalisation()
+        );
     }
 
     private void verifyChildTurnsOneEmailNotificationSentWhenChildTurnsFourOneFirstDayfNextCycle(PaymentCycle paymentCycle, EmailMessagePayload payload) {
         assertThat(payload.getEmailType()).isEqualTo(EmailType.CHILD_TURNS_ONE);
         assertThat(payload.getClaimId()).isEqualTo(paymentCycle.getClaim().getId());
         assertEmailPayloadWhenChildTurnsOneOnFirstDayOfNextCycle(
-                payload.getEmailPersonalisation(),
-                paymentCycle.getCycleEndDate().plusDays(1));
+                payload.getEmailPersonalisation()
+        );
     }
 
     private void verifyChildTurnsFourEmailNotificationSentWhenChildTurnsFourInFirstWeekOfNextCycle(PaymentCycle paymentCycle, EmailMessagePayload payload) {
         assertThat(payload.getEmailType()).isEqualTo(EmailType.CHILD_TURNS_FOUR);
         assertThat(payload.getClaimId()).isEqualTo(paymentCycle.getClaim().getId());
         assertEmailPayloadWhenChildTurnsFourInFirstWeekOfNextCycle(
-                payload.getEmailPersonalisation(),
-                paymentCycle.getCycleEndDate().plusDays(1));
+                payload.getEmailPersonalisation()
+        );
     }
 
     private void verifyChildTurnsOneEmailNotificationSentWhenChildTurnsOneInFirstWeekOfNextCycle(PaymentCycle paymentCycle, EmailMessagePayload payload) {
         assertThat(payload.getEmailType()).isEqualTo(EmailType.CHILD_TURNS_ONE);
         assertThat(payload.getClaimId()).isEqualTo(paymentCycle.getClaim().getId());
         assertEmailPayloadWhenChildTurnsOneInFirstWeekOfNextCycle(
-                payload.getEmailPersonalisation(),
-                paymentCycle.getCycleEndDate().plusDays(1));
+                payload.getEmailPersonalisation()
+        );
     }
 
-    private void assertEmailPayloadWhenChildTurnsFourOnFirstDayOfNextCycle(Map<String, Object> emailPersonalisation,
-                                                                           LocalDate nextPaymentDate) {
-        assertThat(emailPersonalisation).containsOnly(
-                entry("First_name", VALID_FIRST_NAME),
-                entry("Last_name", VALID_LAST_NAME),
-                entry("payment_amount", "£24.80"),
-                entry("pregnancy_payment", ""),
+    private void assertEmailPayloadWhenChildTurnsFourOnFirstDayOfNextCycle(Map<String, Object> emailPersonalisation) {
+        assertThat(emailPersonalisation).contains(
                 entry("children_under_1_payment", "\n* £24.80 for children under 1"),
                 entry("children_under_4_payment", ""),
-                entry("multiple_children", false),
-                entry("next_payment_date", DATE_FORMATTER.format(nextPaymentDate)),
-                entry("regular_payment", "£24.80")
+                entry("multiple_children", false)
         );
+        assertThat(emailPersonalisation).containsAllEntriesOf(COMMON_EMAIL_MAP);
```

Comment:
UpcomingBirthdayEmailHanlder is now adding to a common email map rather than creating one from scratch. Therefore the assertions now check for the presence of the added fields and the common map returned by the EmailMessagePayloadFactory.